README.md

```diff
@@ -489,12 +489,12 @@ under which circumstances:
 | HEAD |  |   |  Read |
 | GET |  |   |  Read |
 | POST | /foo/ | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ gets created | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
-| PATCH | /foo/bar | patch instructions contain deletions | Write on /foo/bar |
+| PUT | /foo/bar | /foo/ exists, will contain /foo/bar | Append or Write on /foo/ + Write on resulting resource |
+| PUT | /foo/bar | /foo/ gets created, will contain /foo/bar | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
```

Comment:
This interaction (recursive create of intermediate directories) is not defined in the specification. If we are describing this as an example, it should be clear whether this is an expected interaction.

Reply:
True! This implicitly ties WAC to Solid which we don't want. We should say 'For instance like when you're using WAC as part of Solid which has `mkdir -p` behaviour', and then link to https://github.com/solid/solid-spec/blob/master/api-rest.md#http-put-to-create